app/forms/providers/application_merits_task/involved_child_form.rb

```diff
@@ -0,0 +1,64 @@
+module Providers
+  module ApplicationMeritsTask
+    class InvolvedChildForm
+      include BaseForm
+      form_for ::ApplicationMeritsTask::InvolvedChild
+
+      BASE_ATTRIBUTES = %i[full_name].freeze
+
+      MODEL_ATTRIBUTES = BASE_ATTRIBUTES + %i[date_of_birth].freeze
+
+      ATTRIBUTES = BASE_ATTRIBUTES + %i[date_of_birth_1i date_of_birth_2i date_of_birth_3i].freeze
+
+      SCOPE = 'activemodel.errors.models.application_merits_task.involved_child.attributes'.freeze
+
+      attr_accessor(*ATTRIBUTES)
+      attr_writer :date_of_birth
+
+      validates :full_name, presence: true
+      validates :date_of_birth, presence: true
+      validates(
+        :date_of_birth,
+        date: {
+          not_in_the_future: true,
+          earliest_allowed_date: { date: '2000-01-01' }
```

Comment:
Is this the actual earliest allowed date? This allows for someone who is 21 years old. We may need to make this date dynamic based on the current date and what we consider a child.